dvc/command/get.py

```diff
@@ -78,6 +78,7 @@ def add_parser(subparsers, parent_parser):
     get_parser.add_argument(
         "--show-url",
         action="store_true",
-        help="Returns path/url to the location in remote for specified path",
+        help="Just print the storage location (URL) the target data would be "
```

Comment:
**Just** feels weird in help text. How about:
```suggestion
        help="Prints the storage location (URL) the target data would be "
```

Reply:
"Just" means that the command won't do anything else i.e. it won't download the target.

Reply:
can we use `Only` instead ...or `print ... and exit`? I was confused with Just either

Reply:
OK using "print and exit" but I think it's just a confusing option in general. Hopefully it will be more intuitive when it's moved to `dvc list`.

Reply:
Done in 6364c9ee.

Reply:
We can be even more explicit - `Skip downloading, print ... and exit`